scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -674,6 +738,39 @@
     (is (= (-> adapter .portRanges first .getBeg) 31000))
     (is (= (-> adapter .portRanges first .getEnd) 32000))))
 
+(deftest test-virtual-machine-lease-adapter-k8s
+  ;; ensure that the VirtualMachineLeaseAdapter can successfully handle an offer from Kubernetes.
+  (let [
+        when (System/currentTimeMillis)
```

Comment:
This should be on the previous line.

Nitpick: I would also favor something like `now` instead of `when` since `when` is a Clojure built-in.

Reply:
modified both